### PR TITLE
[SPARK-19714][ML] Bucketizer.handleInvalid docs improved

### PR DESCRIPTION
**Pull request by wojtek-szymanski**
## What changes were proposed in this pull request?

Docs for `Bucketizer.handleInvalid` has been enhanced with more detailed information about supported and not supported types of invalid entries.

## How was this patch tested?

N/A, just dosc update.

### CODE REVIEW

File: mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala [hunk 1]
```diff
@@ -78,16 +78,19 @@ final class Bucketizer @Since("1.4.0") (@Since("1.4.0") override val uid: String
   def setOutputCol(value: String): this.type = set(outputCol, value)
 
   /**
-   * Param for how to handle invalid entries. Options are 'skip' (filter out rows with
-   * invalid values), 'error' (throw an error), or 'keep' (keep invalid values in a special
-   * additional bucket).
+   * Param for how to handle invalid entries containing either NaN or null values.
```
Comment (MLnick) [inline mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala]: This is actually not true currently - `null` is not handled as an invalid value - see https://issues.apache.org/jira/browse/SPARK-19781

Comment (MLnick) [inline mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala]: I guess once #17123 maybe merged it will hold true

**Comment by MLnick** [comment]
ok to test

File: mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala (hunk 1 above)
Comment (wojtek-szymanski) [inline mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala]: You are right. I will update docs with null values support once #17123 gets merged.

**Comment by wangyum** [comment]
It seems has improved:
https://github.com/apache/spark/blob/v2.4.0/mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala#L91-L104

**Comment by srowen** [comment]
True, though the sentence that is added in this PR isn't added. I can just add it to close the loop here. This PR can be closed , as it seems abandoned

**Pull request closed without merging**